### ISSUE TITLE
BUG when output columns are chosen

### ISSUE THREAD

**Issue by zchmielewska** [acturtle/cashflower#241]
settings.py
```
"OUTPUT_COLUMNS": ["pv_net_cf"],
```

error msg
```
ValueError: Shape of passed values is (721, 1), indices imply (721, 23)
```

**Comment by zchmielewska** [comment]
```
if len(self.settings["OUTPUT_COLUMNS"]) == 0:
    output_columns = [variable.name for variable in self.variables]
else:
    output_columns = self.settings["OUTPUT_COLUMNS"]
```

line 268 cashflow.py

**Comment by zchmielewska** [comment]
solved in d70aa54730f71dcc35993b59d114473f47cf0cc8
